Gemfile

```diff
@@ -18,4 +18,5 @@ group :development do
   gem "puppet-blacksmith"
   gem "nokogiri", "~> 1.6.0"
   gem "guard-rake"
+  gem "xmlrpc"
```

Comment:
Otherwise I kept getting:

```
Could not autoload puppet/type/package: Could not autoload puppet/provider/package/pip: cannot load such file -- xmlrpc/client
```

when running the tests.